packages/mainsail/source/fee.service.ts

```diff
@@ -50,25 +56,13 @@ export class FeeService extends Services.AbstractFeeService {
 		return BigNumber.ZERO;
 	}
 
-	#transform(type: string, typeGroup: number, staticFees: object, dynamicFees: object): Services.TransactionFee {
-		const dynamicFee = (dynamicFees[typeGroup] ?? staticFees[typeGroup])[type] ?? "0";
-		let minimumFee = this.bigNumberService.make(dynamicFee?.min ?? "0");
-		let averageFee = this.bigNumberService.make(dynamicFee?.avg ?? "0");
-		const maximumFee = this.bigNumberService.make(staticFees[typeGroup][type] ?? "0");
-
-		if (type === "multiPayment") {
-			minimumFee = maximumFee;
-			averageFee = maximumFee;
-		}
-
+	#transform(dynamicFees: Fees): Services.TransactionFee {
 		return {
-			avg: averageFee.isGreaterThan(maximumFee) ? maximumFee : averageFee,
-			isDynamic:
-				this.configRepository.get<string>("network.transactions.fees.type") !== "static" &&
-				type !== "multiSignature",
-			max: maximumFee,
-			min: minimumFee.isGreaterThan(maximumFee) ? maximumFee : minimumFee,
-			static: maximumFee,
+			avg: BigNumber.make(dynamicFees?.avg ?? "0").divide(GWEI_MULTIPLIER),
```

Comment:
Couldn't use `formatUnits` here as the return type requires `BigNumber`, not `string`